src/components/GameDetailsScreen.js

```diff
@@ -9,9 +9,17 @@ import {
   Pressable,
 } from "react-native";
 
-const GameDetailsScreen = ({ game, setModalVisible }) => {
+const GameDetailsScreen = ({ game, myGames, addGame, removeGame, setModalVisible }) => {
+  const [hasGame, setHasGame] = useState(false);
+
   const { title, image, genres, consoles } = game;
 
+  useEffect(() => {
```

Comment:
Is this where the `includes` wouldn't work? If so, I think it is because the game is an object and the title is a key.

Reply:
Yea I had `if (myGames.includes(game))` while console.logging both the game and myGames. I could literally see it in the array but for some reason... no work =[